gensim/test/test_ensemblelda.py

```diff
@@ -59,17 +59,17 @@ def test_elda(self):
         assert len(elda.ttda) == NUM_MODELS * NUM_TOPICS
         self.assert_ttda_is_valid(elda)
 
-    def test_backwards_compatibility_with_persisted_model(self):
-        elda = self.get_elda()
-
-        # compare with a pre-trained reference model
-        loaded_elda = EnsembleLda.load(datapath('ensemblelda'))
-        np.testing.assert_allclose(elda.ttda, loaded_elda.ttda, rtol=RTOL)
-        atol = loaded_elda.asymmetric_distance_matrix.max() * 1e-05
-        np.testing.assert_allclose(
-            elda.asymmetric_distance_matrix,
-            loaded_elda.asymmetric_distance_matrix, atol=atol,
-        )
+    # REMOVING THE TEST AS NEW MODELS INITIALIZATIONS WILL BE DIFFERENT FROM PREVIOUS VERSION'S
```

Comment:
Hm. That's tricky. Commenting a test is not a good solution.

If we make such an abrupt compatibility break, we should either

1) Update the pre-trained reference model.
2) Have `load()` replace the affected attributes, transparently.